documentation/prerequisites.md

```diff
@@ -7,13 +7,15 @@ Following is the list of prerequisites for running Kubernetes with vSphere Cloud
 * We recommend Kubernetes version 1.6.5 and vSphere version 6.0.x
```

Comment:
Also fixing this line as ...
"We recommend Kubernetes version 1.6.5 and above with vSphere version 6.0.x"